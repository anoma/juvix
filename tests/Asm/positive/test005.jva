-- Tracing IO

function main () {
  push 1;
  trace;
  push 2;
  trace;
  trace;
  push "seven";
  trace;
  pop;
  pop;
  pop;
  push unit;
  trace;
  pop;
  push void;
  trace;
  ret;
}
