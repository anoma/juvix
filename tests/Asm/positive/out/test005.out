1
2
2
seven
unit
void
